tmt/steps/provision/mock.py

```diff
@@ -20,6 +20,12 @@
 from tmt.utils import Command, OnProcessEndCallback, OnProcessStartCallback, Path, ShellScript
 from tmt.utils.wait import Waiting
 
+MOCK_PIPE_STEM: str = 'srv/tmt-mock'
+MOCK_PIPE_STDOUT_STEM: str = f'{MOCK_PIPE_STEM}/stdout'
+MOCK_PIPE_STDERR_STEM: str = f'{MOCK_PIPE_STEM}/stderr'
+MOCK_PIPE_RETURNCODE_STEM: str = f'{MOCK_PIPE_STEM}/returncode'
+MOCK_PIPE_FILESYNC_STEM: str = f'{MOCK_PIPE_STEM}/filesync'
```

Comment:
I think you should make these `tmt._compat.pathlib.Path` and then selectively use `Path.unrooted` to get rid of the `/`. Afaict the unrooted version is much less common then the rooted one, so you can avoid a rooting construction in most places.